core/test/results/sample_v2.json

```diff
@@ -5831,6 +5831,59 @@
       "description": "Run the [Structured Data Testing Tool](https://search.google.com/structured-data/testing-tool/) and the [Structured Data Linter](http://linter.structured-data.org/) to validate structured data. [Learn more about Structured Data](https://developer.chrome.com/docs/lighthouse/seo/structured-data/).",
       "score": null,
       "scoreDisplayMode": "manual"
+    },
+    "bf-cache": {
+      "id": "bf-cache",
+      "title": "Page prevented back/forward cache restoration",
+      "description": "Many navigations are performed using the back/forward buttons. The back/forward cache can speed up these return navigations. [Learn more about the back/forward cache](https://web.dev/bfcache/)",
+      "score": 0,
+      "scoreDisplayMode": "binary",
+      "displayValue": "1 actionable failure reason",
+      "details": {
+        "type": "table",
+        "headings": [
+          {
+            "key": "reason",
+            "valueType": "text",
+            "subItemsHeading": {
+              "key": "frameUrl",
+              "valueType": "url"
+            },
+            "label": "Failure reason"
+          },
+          {
+            "key": "failureType",
+            "valueType": "text",
+            "label": "Failure type"
+          }
+        ],
+        "items": [
+          {
+            "reason": "The page has an unload handler in the main frame.",
+            "failureType": "Actionable",
+            "subItems": {
+              "type": "subitems",
+              "items": [
+                {
+                  "frameUrl": "http://localhost:10200/dobetterweb/dbw_tester.html"
+                }
+              ]
+            }
+          },
+          {
+            "reason": "Internal error.",
+            "failureType": "Not actionable",
```

Comment:
Hmm looks like it could be one of a few different errors:
https://github.com/GoogleChrome/lighthouse/blob/80709e6c7dd126af3af34d75985a79fbcf7f429e/core/lib/bfcache-strings.js#L568-L576